*: add tidb_enable_stats_owner (#19109)
pingcap/docs: tidb-configuration-file.md

- The value of this configuration will initialize the value of the system variable [`tidb_enable_ddl`](/system-variables.md#tidb_enable_ddl-new-in-v630)
- Before v6.3.0, this configuration is set by `run-ddl`.

### `tidb_enable_stats_owner` <span class="version-mark">New in v8.4.0</span>

> - This configuration controls whether the corresponding TiDB instance can become a stats owner or not.

Reviewer: ```suggestion
- This configuration controls whether the corresponding TiDB instance whether the corresponding TiDB instance can run [automatic statistics update](/statistics.md#automatic-update) tasks.
```

Reviewer: @hawkingrei PTAL

- Default value: `true`

> - Possible values: `OFF`, `ON`

Reviewer: @hawkingrei PTAL

> - This configuration controls whether the corresponding TiDB instance whether the corresponding TiDB instance can run [automatic statistics update](/statistics.md#automatic-update) tasks.

Reviewer 2: ```suggestion
- This configuration controls whether the corresponding TiDB instance can run [automatic statistics update](/statistics.md#automatic-update) tasks.
```

> - The value of this configuration will initialize the value of the system variable [`tidb_enable_stats_owner`](/system-variables.md#tidb_enable_stats_owner-new-in-v840)

Reviewer 2: ```suggestion
- The value of this configuration will initialize the value of the system variable [`tidb_enable_stats_owner`](/system-variables.md#tidb_enable_stats_owner-new-in-v840).
```
